x-pack/plugins/integration_assistant/public/common/lib/api_parsers.ts

```diff
@@ -13,13 +13,20 @@ import type { EpmPackageResponse } from './api';
  * TODO: Return the package name from the fleet API: https://github.com/elastic/kibana/issues/185932
  */
 export const getIntegrationNameFromResponse = (response: EpmPackageResponse) => {
-  const ingestPipelineName = response.response?.[0]?.id;
-  if (ingestPipelineName) {
-    const match = ingestPipelineName.match(/^.*-([a-z\d_]+)\..*-([\d\.]+)$/);
-    const integrationName = match?.at(1);
-    const version = match?.at(2);
-    if (integrationName && version) {
-      return `${integrationName}-${version}`;
+  const items = response?.items?.map((item) => item);
```

Comment:
If you want, there is a much smaller version way to write this, though a bit harder to read :) 

`response?.items?.find(item => item.type === 'ingest_pipeline')?.id?.match(/^.*-([a-z\d_]+)\..*-([\d\.]+)-*([a-z]*)$/)?.[1:3].join('-');`

You could also do this to get a few less lines:
`match = response?.items?.find(item => item.type === 'ingest_pipeline')?.id?.match(/^.*-([a-z\d_]+)\..*-([\d\.]+)-*([a-z]*)$/)`

Reply:
Maybe extracting regex to const will make this more clean

Reply:
Sure, just wanted to show an exmaple on how it could mostly be done without the nested iterations 👍